.github/workflows/storage.yml

```diff
@@ -134,8 +134,8 @@ jobs:
           quickstart-ios/storage/GoogleService-Info.plist "$plist_secret"
     - name: Xcode
       run: sudo xcode-select -s /Applications/${{ matrix.xcode }}.app/Contents/Developer
-    - name: Test objc quickstart
-      run: ([ -z $plist_secret ] || scripts/third_party/travis/retry.sh scripts/test_quickstart.sh Storage true)
+    # - name: Test objc quickstart
+    #   run: ([ -z $plist_secret ] || scripts/third_party/travis/retry.sh scripts/test_quickstart.sh Storage true)
```

Comment:
Can this be removed entirely?

Reply:
This is hopefully a short-term GHA flake.  There is a better comment around this in `main` that will get merged at the next merge.